airgun/views/host_new.py

```diff
@@ -93,7 +93,23 @@ class RepositorySets(Tab):
 
     @View.nested
     class Ansible(Tab):
-        pass
+        ROOT = './/div[contains(@class, "ansible-host-details")]'
+
+        @View.nested
+        class RolesTab(Tab):
+            ROOT = ""
```

Comment:
No need to set this, the `Tab` class implements a locator based on the name of the class

Reply:
https://github.com/RedHatQE/widgetastic.patternfly4/blob/master/src/widgetastic_patternfly4/tabs.py#L6

So, the 'right' way to do it is to set `TAB_NAME` within the tab class, or to name the tab class according to the displayed tab in the UI (comment on TAB_NAME).

No need to set ROOT then, its defined in the `Tab` class and is based on `TAB_NAME`.

The other tabs in this module are using ROOT directly, which isn't ideal, but maybe is necessary if the DOM is messed up? I'd try with TAB_NAME first, and if that works try to apply it to all `Tab` based classes in the view.